src/themes/DarkTheme.js

```diff
@@ -67,4 +72,35 @@ export default {
   loader: {
     backgroundColor: colors.brand,
   },
+
+  dropdown: {
+    backgroundColor: colors.tuna,
+    shadowColor: shadows.shadowColor,
+    titleColor: colors.white,
+    borderColor: colors.mako,
+    borderColorLight: colors.shark,
```

Comment:
Why a single element has 2 border colors (`borderColor`, `borderColorLight`)?
Shouldn't it be something like `borderColorHover`?